tad/services/tasks.py

```diff
@@ -49,7 +49,7 @@ def move_task(
         task = self.repository.find_by_id(task_id)
 
         if status.name == "done":
-            self.system_card.title = task.title
+            self.system_card.name = task.title
```

Comment:
Maybe out of scope, but the TaskService now has a (default / demo) SystemCard based on demo properties. It would be nice if the system card for the selected project would be updated here instead.

Reply:
Then again.. the (demo) tasks we have now are not relatable to the the instruments we can select, so maybe this is too unrelated.